agent/consul/client_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"github.com/hashicorp/consul/internal/resource"
```

Comment:
NIT: we need to run `goimports` on this file.

Reply:
Thanks for spotting this out! Configured my editor so that it will run `goimports` files automatically everytime